src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Plot trajectory from inspiral run, concatenating all segments
+
+import logging
+
+import click
+import h5py
+import matplotlib.pyplot as plt
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+from spectre.Visualization.Plot import (
+    apply_stylesheet_command,
+    show_or_save_plot_command,
+)
+
+
+@click.command(name="trajectories")
+@click.argument(
+    "h5_files",
+    nargs=-1,
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, readable=True),
+)
+@click.option(
+    "--subfile-name-aha",
+    "-A",
+    default="ApparentHorizons/ControlSystemAhA_Centers.dat",
+    help=(
+        "Name of subfile containing the apparent horizon centers for object A."
+    ),
+)
+@click.option(
+    "--subfile-name-ahb",
+    "-B",
+    default="ApparentHorizons/ControlSystemAhB_Centers.dat",
+    help=(
+        "Name of subfile containing the apparent horizon centers for object B."
+    ),
+)
+@click.option(
+    "--sample-rate",
+    "-sr",
+    type=int,
+    default=15,
+    show_default=True,
+    help="Downsampling value for 3D plots",
+)
+@click.option(
+    "--figsize",
+    nargs=2,
+    type=float,
+    default=(10.0, 10.0),
+    show_default=True,
+    help="Figure size as width and height in inches",
+)
+@apply_stylesheet_command()
+@show_or_save_plot_command()
+def plot_trajectories_command(
```

Comment:
Move the command below the two functions, just above `main`. That's usually the structure so the functions are defined by the time they are used here.